src/main/resources/plugin.yml

```diff
@@ -0,0 +1,23 @@
+name: AdminTools
+version: @version@
+main: com.frostcraft.admintools.Main
+prefix: Frostcraft-AdminTools 
+description: Admin tools for FrostCraft
+website: frostcraft.com
+commands:
+  fcadmin:
+    description: FrostCraft AdminTools command
+    usage: Use /fcadmin
+    permission: fcadmin.admin
+    permission-message: You do not have permission to use this command.
+  verify:
+    description: authenticate a player to use the fcadmin command.
+    permission: fcadmin.verify
```

Comment:
Why make verify its own permission instead of part of the admin permission?

Reply:
I can change that.